accounts/templates/index.html

```diff
@@ -4,7 +4,9 @@
     {% if user.id  %}
     <p>ログインしました．</p>
     <p>こんにちは，{{ user.username }}さん</p>
+    <a href="{% url 'login' %}">ログアウトする</a>
```

Comment:
MUST: ここは `logout` だと思います

```suggestion
    <a href="{% url 'logout' %}">ログアウトする</a>
```